site/content/documentation/examples.mdx

```diff
@@ -0,0 +1,19 @@
+---
+order: 1
+title: Examples
+---
+
+### FCD3 App Directory example
+
+This example contains a JSON file that can be used as the data source for ComposeUI's FDC3 App Directory implementation. The App Directory contains the [js-chart](https://github.com/fhubi/ComposeUI/tree/main/examples/js-chart) and [js-grid](https://github.com/fhubi/ComposeUI/tree/main/examples/js-datagrid) examples.
```

Comment:
This is your personal fork, not the main repo ;)

Reply:
Fixed.